scraper.rb

```diff
@@ -43,8 +48,9 @@ def scrape_page(page)
 number_pages =  page.at("#ctl00_MainBodyContent_mPagingControl_pageNumberLabel").inner_text.split(" ")[3].to_i
 
 (1..number_pages).each do |no|
-  url = "https://epathway.monash.vic.gov.au/ePathway/Production/Web/GeneralEnquiry/EnquirySummaryView.aspx?PageNumber=#{no}"
-  page = agent.get(url)
+  result_page_extension = "/EnquirySummaryView.aspx?PageNumber=#{no}"
```

Comment:
I recon you could inline this into the next line, it's clear what it is because it's whats added to base_url to make `results_page_url`?